src/screens/Settings/BusinessMapping/BusinessMappingEntity.res

```diff
@@ -1,124 +1,33 @@
-        let res = await updateDetails(url, values, Post)
-        let filteredProfileList =
-          businessProfiles
-          ->Array.filter(businessProfile => businessProfile.profile_id !== profileId)
-          ->Array.concat([res->BusinessProfileMapper.businessProfileTypeMapper])
-
-        setBusinessProfiles(_ => filteredProfileList)
-        showToast(~message="Updated profile name!", ~toastType=ToastSuccess)
-      } catch {
-      | _ => showToast(~message="Failed to update profile name!", ~toastType=ToastError)
-      }
-      setShowModal(_ => false)
-      Nullable.null
-    }
-
-    let businessName = FormRenderer.makeFieldInfo(
-      ~label="Profile Name",
-      ~name="profile_name",
-      ~placeholder=`Eg: Hyperswitch`,
-      ~customInput=InputFields.textInput(),
-      ~isRequired=true,
-    )
-
-    <div className="flex gap-4 items-center">
-      <ToolTip
-        description="Edit profile name"
-        toolTipFor={<Icon
-          name="pencil-alt"
-          size=14
-          className="cursor-pointer"
-          onClick={_ => setShowModal(_ => true)}
-        />}
-        toolTipPosition=ToolTip.Top
-        contentAlign={Left}
-      />
-      <ToolTip
-        description="Copy profile Id"
-        toolTipFor={<Icon
-          name="copy-code"
-          size=20
-          className="cursor-pointer"
-          onClick={_ => {
-            Clipboard.writeText(profileId)
-            showToast(~message="Copied to Clipboard!", ~toastType=ToastSuccess)
-          }}
-        />}
-        toolTipPosition=ToolTip.Top
-        contentAlign={Left}
-      />
-      <Modal
-        key=defaultProfileName
-        modalHeading="Edit Profile name"
-        showModal
-        setShowModal
-        modalClass="w-1/4 m-auto">
-        <Form initialValues={initialValues->JSON.Encode.object} onSubmit>
-          <div className="flex flex-col gap-12 h-full w-full">
-            <FormRenderer.DesktopRow>
-              <FormRenderer.FieldRenderer
-                fieldWrapperClass="w-full"
-                field={businessName}
-                labelClass="!text-black font-medium !-ml-[0.5px]"
-              />
-            </FormRenderer.DesktopRow>
-            <div className="flex justify-end w-full pr-5 pb-3">
-              <FormRenderer.SubmitButton text="Submit changes" buttonSize={Small} />
-            </div>
-          </div>
-        </Form>
-      </Modal>
-    </div>
-  }
-}
-
 type columns =
   | ProfileName
   | ProfileId
-  | Action
+  | ReturnUrl
+  | WebhookUrl
 
-let visibleColumns = [ProfileId, ProfileName, Action]
+let visibleColumns = [ProfileId, ProfileName, ReturnUrl, WebhookUrl]
 
-let defaultColumns = [ProfileId, ProfileName, Action]
+let defaultColumns = [ProfileId, ProfileName, ReturnUrl, WebhookUrl]
 
-let allColumns = [ProfileId, ProfileName, Action]
+let allColumns = [ProfileId, ProfileName, ReturnUrl, WebhookUrl]
```

Comment:
If all three are same we can make them a single variable and assign the same

Reply:
okay